scenes/game/local_game.gd

```diff
@@ -5233,6 +5586,11 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			events += handle_strike_effect(card_id, discard_effect, performing_player)
 		"power_modify_per_buddy_between":
 			performing_player.strike_stat_boosts.power_modify_per_buddy_between += effect['amount']
+		"powerdown_per_armor_used":
+			var power_change = - performing_player.strike_stat_boosts.consumed_armor
+			if power_change < 0:
```

Comment:
i commented when i saw this effect named, but it does look like you could have "amount" in the effect and keep it in the naming pattern of powerup_*

Just makes it a little more reusable later even if we don't actually need it.